gui/main.qml

```diff
@@ -2,345 +2,123 @@ import QtQuick 2.11
-                    Layout.fillWidth: true
-                    Layout.topMargin: 10
-                    height: parent.height/3
-                    width: parent.width
-                }
-
-                ScopeView
-                {
-                    id: flowView
-                    name: "Flow [mL/min]"
-                    // TODO: Are these reasonable lower and upper bounds?
-                    // Source for current value:
-                    // https://www.sciencedirect.com/topics/medicine-and-dentistry/peak-inspiratory-flow
-                    // "Most modern ventilators can deliver flow rates between
-                    // 60 and 120 L/min. "
-                    yMin: -150
-                    yMax: 150
-
-                    color: "green"
-                    Layout.fillHeight: true
-                    Layout.fillWidth: true
-                    Layout.topMargin: 10
-                    height: parent.height/3
-                    width: parent.width
-                }
+        Image {
+            anchors.horizontalCenter: parent.horizontalCenter
+            // width will naturally be selected preserving ratio
+            height: parent.height
+            source: "images/respiraWorksLogoHorizontalTransparent.png"
+            fillMode: Image.PreserveAspectFit
+        }
 
-                ScopeView
-                {
-                    id: tidalVolumeView
-                    name: "Tidal Volume [mL]"
-                    // TODO: Are these reasonable lower and upper bounds?
-                    // Source for current value:
-                    // https://en.wikipedia.org/wiki/Tidal_volume
-                    // "In a healthy, young human adult, tidal volume is
-                    // approximately 500 mL per inspiration or 7 mL/kg of body mass."
-                    // Meaning, 2000 should be enough for a human of ~300kg body mass;
-                    // I don't know whether heavier humans have even larger
-                    // tidal volume.
-                    yMin: 0
-                    yMax: 2000
+    }
 
-                    color: "#ffff00"
-                    Layout.fillHeight: true
-                    Layout.fillWidth: true
-                    Layout.topMargin: 10
-                    height: parent.height/3
-                    width: parent.width
-                }
+    ModeSelectionPopup {
+        id: modeSelectionPopup
+        currentMode: pageStack.currentMode
+        modesModel: pageStack.modesModel
+    }
 
+    StackView {
+        id: pageStack
+        anchors.fill: parent
 
+        property var currentMode: modesModel.get(0)
 
+        property ListModel modesModel: ListModel {
+            ListElement {
+                mode: "command_pressure_mode"
+                title: qsTr("Command Pressure")
+                acronym: "CPP"
             }
+            ListElement {
+                mode: "pressure_assist_mode"
+                title: qsTr("Pressure Assist")
```

Comment:
I think this is Pressure Assist/Control, acronym P A/C. But generally better to confirm with Pallavi or Neel.